content/faq/linux/how-to-use-flatpak/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How do I use the flatpak version of the app?
+description: "Frequently asked questions for Organic Maps application"
+
+taxonomies:
+  faq: ["Linux"]
+
+extra:
+  order: 10
+---
+
+To install OM on linux with flatpak, open the terminal and imput `flatpak install flathub app.organicmaps.desktop`
+
+Once the app is installed you cam use the scroll wheel of your mouse or the controlls on the right menue bar to zoom into the area you wish to navigate in order to download maps for that area. You can also click the "download" icon in the lower right. Once you have downloaded maps for the regions you care about, the app should work even without an internet connection. 
+
+You can hover your mouse over the various icons to see some help text. 
+
+To perform routing and turn by turn navigation, you have a couple of options. if you know the gps cordinates of your starting point and ending point, you can click the settings icon (above the green checkmark) and input the cordinetes of your starting point and destination. To set the starting point on the map itself, click the navigation icon and select "start point", then hold down the shift key and left click on the map. To set the destination, change to "end point" and click a location on the map.
+
+![Setting start point and end point](nav_points.png)
+
+You can click the blue icon directly above settings icon to clear navigation. 
+
+![clear navigation](clear_route.png)
+
+To search for addresses and destinations, click the magnifying glass and enter the address or search term.
+
+To add a location to your bookmarks, hold down the Alt key and right click the place you want to bookmark. The bookmark may not be immediatly visible, to view and manage bookmarks you need to click the red star icon. 
```

Comment:
```suggestion
To bookmark a location, hold down the Alt key and right-click the place you want to bookmark. The bookmark may not be immediately visible, to view and manage bookmarks you need to click the red star icon. 
```

Reply:
What does it mean "may not be immediately visible"? It looks like a bug. Does it always reproduce?